bookshelf/tests/controllersTest.php

```diff
@@ -0,0 +1,232 @@
+<?php
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+namespace Google\Cloud\Samples\Bookshelf;
+require_once __DIR__.'/../vendor/autoload.php';
+
+use Silex\WebTestCase;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+
+/**
+ * Class FakeImageStorage
+ * @package Google\Cloud\Samples\Bookshelf
+ *
+ * Implements ImageStorage in a simple way that we can verify in tests.
+ */
+class FakeImageStorage implements ImageStorage
```

Comment:
I'd like to see this in its own directory, `test/lib/FakeImageStorage.php` ... I just can't get myself to ever like seeing more than one class in a single file